.github/workflows/ci.yml

```diff
@@ -12,15 +12,15 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        node: [14, 16, 18]
+        node-version: [14, 16, 18]
     runs-on: ${{ matrix.os }}
     steps:
       - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
+      - uses: volta-cli/action@v1
         with:
-          node-version: ${{ matrix.node }}
-
-      - run: npm run install
+          node-version: ${{ matrix.node-version }}
+          npm-version: '8.5.0'
+      - run: npm ci
```

Comment:
**Changes:**

* `node` => `node-version`, most repos name the node matrix this, don't know why, but just making it like the rest.
* `npm 8.5.0` - By default Node 14 comes with npm 6, which supports v1 lock files. But this PR includes a V2 lockfile. So we need to use npm 7 or above for that.
* `actions-setup-node` => `volta-cli/action`, This lets us easily set the Node and npm versions in one go, so we can do Node 14, 16, or 18 all with npm 8.5.0
* `npm run install` => `npm ci`, this makes sure we download the exact same dependency versions that were used during development